app/models/training_data/code_tags_sample.rb

```diff
@@ -70,4 +70,7 @@ def next_status
       status
     end
   end
+
+  memoize
+  def code = files.to_a.pluck("code").join('\n')
```

Comment:
Not sure how we want to handle multiple files, so I just concatenated them

Reply:
Yeah - I wasn't sure about this either. I feel like it will make it harder for an LLM to cope if both `c` and `h` files are concatenated but we can experiment. This is fine for now.